### PR TITLE
update CALs data 

### PR DESCRIPTION
**Pull request by einc**
### Summary 

Updating Colleges Data

- [x] update CALs
- [x] small minor req fixes noted in Beta Req Bugs
- [x ] updated comments like disallowTransferCredit that Ben added once his PR is approved (i just added them so I don't forget for now)

### Test Plan 

- [x] test CALS college (in particular, calsPhysicalAndLifeSciencesRequirement is now no longer self check and the 55 CALs credit are double counted)

### CODE REVIEW

**Comment by SamChou19815** [comment]
@einc Ben's PR is in so you can replacing the comments with rule code now. Also does it PR completely replaces this dead branch: https://github.com/cornell-dti/course-plan/tree/updateCollegesData

File: src/requirements/data/colleges/ag.ts [hunk 1]
```diff
@@ -107,10 +100,447 @@ const calsIntroductoryLifeSciencesOrBiologyRequirement: CollegeOrMajorRequiremen
...
   description:
-    '18 credits in at least three disciplines of which six credits must be introductory life sciences/biology and three credits in chemistry or physics and a quantitative literacy course.',
+    '18 credits in at least three disciplines of which six credits must be introductory life sciences/biology and three credits in chemistry or physics',
   source:
     'https://cals.cornell.edu/undergraduate-students/student-services/degree-requirements/graduation-requirements/distribution-requirements',
-  fulfilledBy: 'self-check',
+  checker: includesWithSingleRequirement(
+    'AEM 2100',
+    'AEM 3100',
+    'AEM 3100',
```
Comment (tcho6319) [inline src/requirements/data/colleges/ag.ts:109]: Seems to be a dupe

**Comment by einc** [comment]
> @einc Ben's PR is in so you can replacing the comments with rule code now. Also does it PR completely replaces this dead branch: https://github.com/cornell-dti/course-plan/tree/updateCollegesData

just uncommented, and yes it does. i'll delete that branch now

File: src/requirements/data/colleges/ag.ts [hunk 2]
```diff
@@ -107,10 +100,447 @@ const calsIntroductoryLifeSciencesOrBiologyRequirement: CollegeOrMajorRequiremen
...
+    'BTRY 6840',
+    'CEE 3040',
+    'CHEM 1150',
+    'CHEM 1560',
+    'CHEM 1570',
+    'CHEM 2070',
+    'CHEM 2080',
+    'CHEM 2090',
+    'CHEM 2150',
+    'CHEM 2870',
```
Comment (tcho6319) [inline src/requirements/data/colleges/ag.ts:235]: I think you're missing CHEM 2510
Screen Shot 2021-03-23 at 10 23 27 AM

File: src/requirements/data/colleges/ag.ts [hunk 3]
```diff
@@ -107,10 +100,447 @@ const calsIntroductoryLifeSciencesOrBiologyRequirement: CollegeOrMajorRequiremen
...
+    'CSS 2600',
+    'CSS 3150',
+    'CSS 4050',
+    'CSS 4140',
+    'DSOC 2020',
+    'EAS 1101',
+    'EAS 1109',
+    'EAS 1190',
+    'EAS 1220',
+    'EAS 1310',
```
Comment (tcho6319) [inline src/requirements/data/colleges/ag.ts:266]: Missing EAS 1330
Screen Shot 2021-03-23 at 10 26 48 AM

File: src/requirements/data/colleges/ag.ts [hunk 4]
```diff
@@ -107,10 +100,447 @@ const calsIntroductoryLifeSciencesOrBiologyRequirement: CollegeOrMajorRequiremen
...
+    'ENTOM 4730',
+    'FDSC 2000',
+    'FDSC 2204',
+    'FDSC 4300',
+    'FDSC 4600',
+    'GOVT 2240',
+    'HORT 2204',
+    'HORT 2240',
+    'HORT 3600',
+    'HORT 4260',
```
Comment (tcho6319) [inline src/requirements/data/colleges/ag.ts:328]: Missing HORT 4400
Screen Shot 2021-03-23 at 10 29 27 AM

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Missing some courses and seems that @benjamin-shen 's PR has been merged in, if you'd want to update.

**Review by tcho6319 — APPROVED** [review]
LGTM

**Pull request merged**